### ISSUE TITLE
Start of window cannot be configured in Pipeline API

### ISSUE THREAD

**Issue by olukas** [hazelcast/hazelcast-jet#1432]
In current implementation in Pipeline API there is no option to configure when the window should start. 

It means that it is impossible to set up scenarios like 1-day tumbling window which starts at `00:00:05`.

**Comment by mtopolnik** [comment]
We can easily support this with an extra `offset` parameter which we already had but removed for simplicity.

**Comment by 0x3E6** [comment]
Can we set the `offset` parameter in jet 4.5.3?

**Comment by viliam-durina** [comment]
Nope, and we don't plan it. Could you please describe your use case? Maybe we'll change our mind.
